scripts/compute-basket-weights.ts

```
@@ -0,0 +1,69 @@
+  // 2. Compute and propose weights
+  console.log(`Ingesting metrics and computing weights for period: ${period}`);
+  await ingestMetricsAndProposeWeights(period);
+
+  // 3. Fetch the updated metrics and proposed weights
+  const metrics = await prisma.basketMetrics.findMany({
+    where: { period },
+  });
+
+  const proposedConfigs = await prisma.basketConfig.findMany({
+    // Fetch proposals created recently (today)
+    where: { status: "proposed", effectiveFrom: { gte: new Date(new Date().setHours(0,0,0,0)) } },
```

ESLint

[error] Replace `·status:·"proposed",·effectiveFrom:·{·gte:·new·Date(new·Date().setHours(0,0,0,0))·}` with `⏎······status:·"proposed",⏎······effectiveFrom:·{·gte:·new·Date(new·Date().setHours(0,·0,·0,·0))·},⏎···`

(prettier/prettier)